### ISSUE TITLE
User avatar distortion

### ISSUE THREAD

**Issue by yangm97** [matrix-construct/construct#72]
![image](https://user-images.githubusercontent.com/4412002/55728155-eb36cd00-59e9-11e9-82b0-d68545747b96.png)

**Comment by jevolk** [comment]
That image is distorted because it's not perfectly square. Construct does nothing except pass the original image to the client right now. This will be fixed by implementing #40 but this is really a client bug in Riot. 

The spec says:
> Homeservers **may** generate thumbnails for content uploaded to remote homeservers themselves or may rely on the remote homeserver to thumbnail the content. **Homeservers may return thumbnails of a different size to that requested.** 

Which means that if the server doesn't support thumbnails for the image type (or at all) the client should have a proper fallback. In Riot's case, there's no excuse to not use the browser's facilities for thumbnailing that image. There is nothing we can really do about this except implement #40.

You can leave this issue open to be reaped after #40 or you can just close it for cleanliness since it's already covered; up to you.

**Comment by jevolk** [comment]
Spec sez at https://matrix.org/docs/spec/client_server/r0.4.0.html#get-matrix-media-r0-thumbnail-servername-mediaid

>  method | enum | The desired resizing method. One of: ["crop", "scale"]

If the client requested "crop" this can also be fixed server-side if resizing from #40 is insufficient.
